### ISSUE TITLE
AppVeyor build broken :(

### ISSUE THREAD

**Issue by alaingdl** [gnudatalanguage/gdl#746]
AppVeyor builds are broken. It was not that clear because the results seems to be OK but the compilation it-self is not OK :( 

Maybe it's me, but sorry. (I just added few new keywords after few changes in CMakeLists.txt few weeks ago.

(Is it an easy way to test that locally, without using PR and AppVeyor build ??)

**Comment by slayoo** [comment]
@maynardGK ?

BTW, it's actually possible to remotely log in into the Appveyor machine:
https://www.appveyor.com/docs/how-to/rdp-to-build-worker/

**Comment by maynardGK** [comment]
The Appveyor builds look ok (#745) - problems are with travis. 
(Just looking at the green check marks) Are you having trouble with the product built.  I'll check it out some more.

**Comment by slayoo** [comment]
Travis issue is related with numpy/Python 3.8 - I'll look at it

**Comment by alaingdl** [comment]
It's appears to be OK but if we look in details, eg https://ci.appveyor.com/project/slayoo/gdl/builds/32570973/job/fowdf8ow32whhcc4 the compilation is not done (errors compiling the Cpp code)

**Comment by maynardGK** [comment]
t appears the #ifdef WIN32 logic it screwing it up downstream, in libinit.cpp:
````

  const string spawnKey[]={ "COUNT","EXIT_STATUS","NOSHELL","NULL_STDIN","PID","STDERR","UNIT", //All platforms
#ifdef _WIN32
	  "HIDE", "LOG_OUTPUT", "NOWAIT",};
#else
  "NOTTYRESET","SH",
#endif
  KLISTEND};
 
````

**Comment by alaingdl** [comment]
seems to come from here

https://github.com/gnudatalanguage/gdl/commit/e6359d5ac2132d90f7552431998a767a124e84d0#diff-8140a7b17c8e6a29f2103efabf538dbf

As the message was not clear ...

I would prefer you try to resolve it because you are way better than me for all those _win32 flags !

**Comment by slayoo** [comment]
Travis fixed: https://travis-ci.org/github/gnudatalanguage/gdl/builds/682707621
(solution: don't touch homebrew numpy; update numpy with pip after all other homebrew packages installed; instead of brew install use brew bundle... magick)

**Comment by slayoo** [comment]
Appveyor issue fixed by @maynardGK in #748 
Thanks!
